### PR TITLE
fix(dark-theme): fixes dark theme shadow pf-size-prem

### LINKED ISSUE #4749: Bug - Dark theme shadows - pf-size-prem function not compiling for dark theme
**Describe the problem**
`pf-size-prem()` function not compiling for dark theme

**How do you reproduce the problem?**
Provide steps to reproduce. A [codepen](http://codepen.io) or [codesandbox](https://codesandbox.io/) demonstrating the problem is appreciated.

Screen Shot 2022-03-23 at 11 37 33 AM

### PR DESCRIPTION
**Pull request by mattnolting**
closes #4749

### CODE REVIEW

**Review by evwilkin — APPROVED** [review]
LGTM

File: src/patternfly/themes/dark/scss-variables.scss [hunk 1]
```diff
@@ -1,3 +1,6 @@
+@import "../../sass-utilities/functions"; // functions
+@import "colors"; // dark colors
```
Comment (mcoker) [inline src/patternfly/themes/dark/scss-variables.scss:2]: I might put these in the main dark stylesheet here - https://github.com/patternfly/patternfly/blob/cc6c2257ad05e4e7b5a086bf8ca2d8e3beb993ef/src/patternfly/themes/dark/_patternfly-theme-dark.scss#L1-L9

And mimic the ordering of the existing sass-utilities import order from https://github.com/patternfly/patternfly/blob/main/src/patternfly/sass-utilities/_all.scss. 

It looks like [colors.scss is already imported](https://github.com/patternfly/patternfly/blob/cc6c2257ad05e4e7b5a086bf8ca2d8e3beb993ef/src/patternfly/themes/dark/_patternfly-theme-dark.scss#L4), so I wouldn't think it needs to be imported again?

Comment (mattnolting) [reply src/patternfly/themes/dark/scss-variables.scss:2]: @mcoker updated!

**Review by mcoker — APPROVED** [review]
L🚀TM!

**Pull request merged**